webform_import.module

```diff
@@ -373,14 +380,19 @@ function _webform_import_import($form, $form_state, $file) {
       continue 1;
     }
 
+    // Set defaults
```

Comment:
This can be removed, or it could be changed to `Set defailts.` ending with a period.